cmd/bridge/config.go

```diff
@@ -55,6 +55,8 @@ type Auth struct {
 type Customization struct {
 	Branding             string `yaml:"branding"`
 	DocumentationBaseURL string `yaml:"documentationBaseURL"`
+	CustomProductName    string `yaml:"customProductName"`
```

Comment:
Should we start commenting these?  All of our `api` resources in `openshift/api` are required to have a comment describing each configuration option.  See [console](https://github.com/openshift/api/blob/master/config/v1/types_console.go#L56) as an example.  We prob should do the same for this, if it ever gets promoted we will have to fill it out anyway.

Reply:
I'd rather go through and do it once for all props instead of having only one commented. fwiw, users generally won't set this directly since it's an operator-managed config

Reply:
Do we care about something like this:

```golang
Customization struct {
   Branding: "okd"
   // so its clear, we use `CustomBranding` if it exists, else `Branding`.
   // rather than worrying about 3 top level keys
   CustomBranding: {
      CustomProductName: "foo bar"
      CustomLogoFile: "/path/to/file.png"
   }
}
```
Which could be cleaner:

```golang
Customization struct {
   Branding: "okd"
   // so its clear, we use `CustomBranding` if it exists, else `Branding`.
   // rather than worrying about 3 top level keys
   CustomBranding: {
      ProductName: "foo bar"
      LogoFile: "/path/to/file.png"
   }
}
```
The `cli` flags are fine.  This would just affect the shape of our [operator config](https://github.com/openshift/api/blob/master/operator/v1/types_console.go), as:


```yaml
Customization:
   // built-in
   Branding: okd
   // custom
   CustomProductName: "foo bar"
   CustomLogoFile: "/path/to/file.png"

// vs this seems clearer/less repetitive
Customization:
   // built-in
   Branding: okd
   // custom
   CustomBranding:
      ProductName: "foo bar"
      LogoFile: "/path/to/file.png"
```

If we need more items for `CustomBranding` later, I'd much rather have them collected than scattered all over as top level keys. That said, it is already under `Customization`.

Reply:
Also, lets make sure the flow is clear.  This is `operator` `config` changes, BUT the `configmap` holding the `image` is actually going to be in `openshift-config` (not `openshift-config-managed`), thus the shape I stated above is actually not quite right.   It just needs to be a `configmap` reference. Correct?